doc/theoretical_description_multilabel_classification.rst

```diff
@@ -175,14 +185,17 @@ hypothesis testing. We can express the goal of the procedure as follows:
 In order to find all the parameters :math:`\lambda` that satisfy the above condition, Learn Then Test propose to do the following:
```

Comment:
In order to find all the parameters :math:`\lambda` that satisfy the above condition, the Learn Then Test framework proposes to do the following:

Reply:
```suggestion
In order to find all the parameters :math:`\lambda` that satisfy the above condition, the Learn Then Test framework proposes to do the following:
```

Reply:
done